easybuild/easyconfigs/l/LUMPY/LUMPY-0.2.13-goolf-1.7.20.eb

```diff
@@ -0,0 +1,48 @@
+# This file is an EasyBuild reciPY as per http://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'ConfigureMake'
+
+name = 'LUMPY'
+version = '0.2.13'
+
+homepage = 'https://github.com/arq5x/lumpy-sv'
+description = 'lumpy: a general probabilistic framework for structural variant discovery'
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# no source_urls because the tarball has to be manually generated doing 
+# git clone --recursive https://github.com/arq5x/lumpy-sv.git
+sources = ['%(name)s-%(version)s.tar.gz']
```

Comment:
@pescobar Missing `SHA256` checksum?

Also, seems like there are proper tarballs now, see https://github.com/arq5x/lumpy-sv/releases/download/0.2.13/lumpy-sv-v0.2.13.tar.gz ?

Reply:
@boegel tarballs are provided but don't include all the required sources
https://github.com/arq5x/lumpy-sv/blob/master/.gitmodules